DOC-1573: Updated PowerPaste command documentation (#2287)
tinymce/tinymce-docs: modules/ROOT/partials/commands/powerpaste-cmds.adoc

[cols="1,3",options="header"]
|===
|Command |Description

> |mceInsertClipboardContent |Triggers a paste event to detect Word, Google Docs, or standard HTML content in the clipboard, and use the relevant paste path at the cursor location or over the current selection. The command requires an object with: `+html+` containing the HTML content, or `+text+` containing plain text.

Reviewer: "and use the relevant paste path"

change to:
"and uses the relevant paste path"

Reviewer 2: ```suggestion
|mceInsertClipboardContent |Triggers a paste event at the cursor location or over the current selection. HTML content will be automatically detected as Word, Google Docs, or standard HTML content and use the relevant paste path. The command requires an object with: `+html+` containing the HTML content, or `+text+` containing plain text.
```
